docs/content/migrate/nginx-to-traefik.md

```diff
@@ -0,0 +1,726 @@
+
+1. Deploy Traefik with a new Load Balancer
+2. Update DNS CNAME records to point to the new NLB
+3. Wait for DNS propagation
+4. [Decommission NGINX](#step-7-decommission-nginx-ingress-controller)
+
+### Azure Load Balancer
+
+Azure supports static public IPs for Load Balancers. You can reuse the existing public IP resource.
+
+**Identify Existing Public IP:**
+
+```bash
+# Get the public IP resource name
+az network public-ip list --resource-group <your-resource-group> --query "[?ipAddress=='<your-ip>'].name" -o tsv
+```
+
+Update `traefik-values.yaml`:
+
+```yaml
+service:
+  type: LoadBalancer
+  annotations:
+    service.beta.kubernetes.io/azure-load-balancer-resource-group: "<your-resource-group>"
+  spec:
+    loadBalancerIP: "<your-existing-ip>"
+```
+
+!!! tip "Deleting NGINX Service"
+
+    To successfully reuse the IP, you must delete the NGINX service **before** creating the Traefik service:
+
+    ```bash
+    # Scale down NGINX first
+    kubectl scale deployment -n ingress-nginx ingress-nginx-controller --replicas=0
+
+    # Delete NGINX service to release the IP
+    kubectl delete svc -n ingress-nginx ingress-nginx-controller
+
+    # Install Traefik immediately after
+    helm install traefik traefik/traefik -f traefik-values.yaml
+    ```
+
+### GCP Load Balancer
+
+GCP supports static IPs through reserved IP addresses.
+
+**Reserve or Identify Existing IP:**
+
+```bash
+# List existing static IPs
+gcloud compute addresses list
+
+# Or create a new static IP
+gcloud compute addresses create traefik-ip --region us-central1
+```
+
+Update `traefik-values.yaml`:
+
+```yaml
+service:
+  type: LoadBalancer
+  spec:
+    loadBalancerIP: "<your-static-ip>"
+```
+
+### Other Cloud Providers
+
+For other cloud providers, check their Kubernetes LoadBalancer documentation:
+
+- **DigitalOcean:** Supports `loadBalancerIP` with floating IPs
+- **Linode:** Supports `loadBalancerIP` specification
+- **Bare Metal:** Use MetalLB with IP address pools
+
+---
+
+## Step 3: Install Traefik
+
+With your values file configured, install Traefik:
+
+```bash
+# Install Traefik in a dedicated namespace
+kubectl create namespace traefik
+
+helm install traefik traefik/traefik \
+  --namespace traefik \
+  --values traefik-values.yaml
+```
+
+### Verify Installation
+
+```bash
+# Check Traefik pods are running
+kubectl get pods -n traefik
+
+# Check the service has the correct LoadBalancer IP
+kubectl get svc -n traefik traefik
+
+# Check Traefik logs
+kubectl logs -n traefik deployment/traefik
+```
+
+Expected output:
+
+```bash
+NAME      TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)
+traefik   LoadBalancer   10.0.123.45     <your-lb-ip>     80:<nodeport>/TCP,443:<nodeport>/TCP
+```
+
+---
+
+## Step 4: Test with Sample Application
+
+Before migrating production workloads, test Traefik with a sample application using the `traefik/whoami` image.
```

Comment:
I'm not sure about this step. Since Nginx is already uninstalled, why test before migrating production workloads? Those have already been migrated by the switch. I could imagine such a test when Traefik is installed _besides_ Nginx, then upgraded with an ingress class to be the replacement, but here, we're after the main action.